### PR TITLE
[BEAM-3998] Futurize examples subpackage

### PR DESCRIPTION
**Pull request by Fematich**
This pull request prepares the examples subpackage for Python 3 support. This PR is part of a series in which all subpackages will be updated using the same approach.
This approach has been documented [here ](https://docs.google.com/document/d/1xDG0MWVlDKDPu_IW9gtMvxi2S9I0GB0VDTkPhjXT0nE/edit#heading=h.5vrnrgpkrk89) and the first pull request in the series (Futurize coders subpackage) demonstrating this approach can be found at #5053.

R: @aaltay @tvalentyn @RobbeSneyders

### CODE REVIEW

File: sdks/python/apache_beam/examples/snippets/snippets.py [hunk 1]
```diff
@@ -1048,7 +1055,7 @@ def to_entity(content):
     entity = entity_pb2.Entity()
     googledatastore.helper.add_key_path(entity.key, kind, str(uuid.uuid4()))
     googledatastore.helper.add_properties(entity,
-                                          {'content': six.text_type(content)})
+                                          {'content': unicode(content)})
```
Comment (tvalentyn) [inline sdks/python/apache_beam/examples/snippets/snippets.py]: Several other modules in the `examples` package also use six, let's clean them up. 
See: https://github.com/apache/beam/search?q=six.text_type&unscoped_q=six.text_type.

**Comment by tvalentyn** [comment]
R: @charlesccychen

**Comment by tvalentyn** [comment]
Thanks, @Fematich. Did you push the updated commit? I still see `six` removed only in one location, and still see `//` used in a few places I commented out where I think '\' should be used.

**Comment by Fematich** [comment]
Sorry for that!  Apparently I didn't push my latest commit, fixed now

**Pull request merged**